template/sbt.go

```diff
@@ -58,7 +58,7 @@ services:
       {{.ScratchVolumes}}
   {{end}}
   sbt: &sbt
-    image: arch-docker.eng.lancope.local:5000/sbt
+    image: hseeberger/scala-sbt
```

Comment:
I thought we were not going to use a non-official image here. I know we have to remove the arch-docker one, but I would rather set this to something that won't work, and tell the user to use something else, as opposed to using something that we haven't vetted. I would argue that we set it to `library/busybox`, just to have something there.